<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphwindow</class>
 <widget class="QMainWindow" name="Graphwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>791</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>791</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">text-align: center;
text-decoration: none;
display: inline-block;
font-size: 25px;</string>
       </property>
       <property name="text">
        <string>Function</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="benter">
       <property name="styleSheet">
        <string notr="true">background-color: #e7e7e7; 
color: black;
border: 2px solid black;
padding: 5px 32px;
text-align: center;
text-decoration: none;
display: inline-block;
font-size: 24px;
border-radius: 5px</string>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fun">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white; 
color: black;
border: 2px solid black;
padding: 2px;
text-decoration: none;
display: inline-block;
font-size: 24px;
border-radius: 5px</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>791</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">text-align: center;
text-decoration: none;
display: inline-block;
font-size: 25px;</string>
       </property>
       <property name="text">
        <string>xMin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="xMin">
       <property name="styleSheet">
        <string notr="true">background-color: white; 
color: black;
border: 2px solid black;
padding: 2px;
text-decoration: none;
display: inline-block;
font-size: 24px;
border-radius: 5px</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">text-align: center;
text-decoration: none;
display: inline-block;
font-size: 25px;</string>
       </property>
       <property name="text">
        <string>xMax</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="xMax">
       <property name="styleSheet">
        <string notr="true">background-color: white; 
color: black;
border: 2px solid black;
padding: 2px;
text-decoration: none;
display: inline-block;
font-size: 24px;
border-radius: 5px</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">text-align: center;
text-decoration: none;
display: inline-block;
font-size: 25px;</string>
       </property>
       <property name="text">
        <string>step</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="step">
       <property name="styleSheet">
        <string notr="true">background-color: white; 
color: black;
border: 2px solid black;
padding: 2px;
text-decoration: none;
display: inline-block;
font-size: 24px;
border-radius: 5px</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">text-align: center;
text-decoration: none;
display: inline-block;
font-size: 25px;</string>
       </property>
       <property name="text">
        <string>yMin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="yMin">
       <property name="styleSheet">
        <string notr="true">background-color: white; 
color: black;
border: 2px solid black;
padding: 2px;
text-decoration: none;
display: inline-block;
font-size: 24px;
border-radius: 5px</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">text-align: center;
text-decoration: none;
display: inline-block;
font-size: 25px;</string>
       </property>
       <property name="text">
        <string>yMax</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="yMax">
       <property name="styleSheet">
        <string notr="true">background-color: white; 
color: black;
border: 2px solid black;
padding: 2px;
text-decoration: none;
display: inline-block;
font-size: 24px;
border-radius: 5px</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="bclear">
       <property name="styleSheet">
        <string notr="true">background-color: #e7e7e7; 
color: black;
border: 2px solid black;
padding: 5px 32px;
text-align: center;
text-decoration: none;
display: inline-block;
font-size: 24px;
border-radius: 5px</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
